lib/fml_forms/form.rb

```diff
@@ -1,6 +1,6 @@
 module FML
   class Form
-    attr_reader :form, :title, :version, :fieldsets, :fields, :bodySystem, :dependent
+    attr_reader :form, :title, :id, :version, :fieldsets, :fields, :bodySystem, :dependent
```

Comment:
parenthetical, but FML should be generic for forms, body system is something specific to CUI, and should just be an attribute.  I think the same is true for dependent.